### ISSUE TITLE
Cancel a favorite recording

### ISSUE THREAD

**Issue by 49studebaker** [google/sagetv#217]
I have a TV series set as a favorite. Part of the in progress recording is missing because of a weak TV signal. Is there any way to stop the in progress recording?

**Comment by Narflex** [comment]
Not really. Marking it watched or don't like might stop it...but I'm not
sure for in progress recordings.

Jeff Kardatzke

**Comment by jriskin** [comment]
My usual fix is start watching it live, mark it as watched, then click stop. Then you have to go back and mark it as unwatched to get it to record again, then delete it. It sounds complicated, but it's pretty easy...
